packages/core/src/payment/strategies/stripe-upe/stripe-upe.ts

```diff
@@ -194,7 +194,12 @@ interface ShippingDefaultValues {
 }
 
 interface CustomerDefaultValues {
+    mode?: string;
     email: string;
+    allowedCountries?: string[];
+    display?: {
+        name: string;
```

Comment:
This can also be typed better https://stripe.com/docs/js/elements_object/create_address_element#address_element_create-options-display-name

Reply:
added to the enum created for last comment